peachjam/js/components/FindDocuments/index.vue

```diff
@@ -475,7 +498,20 @@ export default {
         if (params.has('date_to')) this.advancedFields.date.date_to = params.get('date_to');
         advancedSearchFields.forEach(key => {
           if (!params.has(key)) return;
-          this.advancedFields[key] = params.get(key);
+          const fieldValue = params.get(key);
+          const splitValue = fieldValue.match(/[^\s"']+|['"][^'"]*["']+/g);
+
+          splitValue.forEach((value, index) => {
```

Comment:
This isn't quite the right place to do this. The inputs the user provided to the advanced search boxes don't change. Instead, when generating the query string to send to the server, we transform the advanced search fields.

In the case where the user has entered boolean terms for the "all fields" field (not yet in your UI), then the `search` param would be updated to produce, for example, `foo -"bar baz"`.

If the user has entered boolean terms for the "title" field, then the `search__title` param would be updated appropriately, etc.